audio_helper/src/openxtargo.h

```diff
@@ -0,0 +1,78 @@
+//
+// Copyright (c) 2015 Assured Information Security, Inc
+//
+// Dates Modified:
+//  - 4/8/2015: Initial commit
+//    Rian Quinn <quinnr@ainfosec.com>
```

Comment:
This "Dates Modified" section should be dropped. It's not accurate with this change and git history is a better and automatic source of modification history anyway.